dWorldServer/WorldServer.cpp

```diff
@@ -234,6 +235,30 @@ int main(int argc, char** argv) {
 		Game::physicsWorld = &dpWorld::Instance(); //just in case some old code references it
 		dZoneManager::Instance()->Initialize(LWOZONEID(zoneID, instanceID, cloneID));
 		g_CloneID = cloneID;
+
+		// pre calculate the FDB checksum
+		if (bool(std::stoi(Game::config->GetValue("check_fdb")))) {
+				std::ifstream fileStream;
+				fileStream.open ("res/CDServer.fdb", std::ios::binary | std::ios::in);
```

Comment:
Remove the space here `open (`

Reply:
VS won't stop doing this, it is really annoying, sorry